parametric/test_headers_datadog.py

```diff
@@ -1,12 +1,24 @@
+from typing import Any
+
 import pytest
 
 from parametric.spec.trace import SAMPLING_PRIORITY_KEY, ORIGIN
 from parametric.spec.trace import span_has_no_parent
 from parametric.utils.headers import make_single_request_and_get_inject_headers
 from parametric.utils.test_agent import get_span
 
+parametrize = pytest.mark.parametrize
+
+
+def enable_datadog_propagator() -> Any:
+    env = {
+        "DD_TRACE_PROPAGATION_STYLE_EXTRACT": "datadog",
+        "DD_TRACE_PROPAGATION_STYLE_INJECT": "datadog",
+    }
+    return parametrize("library_env", [env])
+
 
-@pytest.mark.skip_library("golang", "not implemented")
+@enable_datadog_propagator()
```

Comment:
Is this change made because the default isn't `datadog` for go? If not, can we revert this change as it would be introducing an unnecessary change.

(also the logic of the tests would be slightly different, we'd be testing the explicit datadog behaviour rather than the default of each library)

Reply:
The default now it 'tracecontext,Datadog'. But let me double-check